### DISCUSSION TITLE
Block Explorer claimed as Open Source, but is not.

### DISCUSSION THREAD

**Discussion by turbotas** [Chia-Network/chia-blockchain#8607] [Ideas]
this message on the bottom of the "Farm" screen of the GUI seems inaccurate: "Want to explore Chia’s blocks further? Check out Chia Explorer built by an open source developer."  The block explorer is a commercial site and the developer is not named nor identified and the explorer is not open source,  Could we have a little more honesty?

**Comment by loppefaaret** [comment]
Thank you very much for bringing this to us, let me just go through this right now, and see what we can do about it!

**Comment by BrandtH22** [comment]
@turbotas , the newest version of chia removes this note. Please download it from here: https://www.chia.net/downloads/

Since this discussion has been deprecated with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.

**Reply by turbotas** [reply]
Reasonable fix.  Shame that it took two years ;-)
